### PR TITLE
Sept.md

### PR DESCRIPTION
**Pull request by wanda-carlson**
:)

### CODE REVIEW

**Comment by wanda-carlson** [comment]
Hi from Github

**Comment by wanda-carlson** [comment]
Hi frrom Slack

**Review by wanda-carlson — COMMENTED** [review]
message

**Review by wanda-carlson — COMMENTED** [review]
Hi There!

**Review by wanda-carlson — COMMENTED** [review]
minor changes please

File: README.md [hunk 1]
```diff
@@ -31,3 +31,4 @@ Test4
 111
 
 Test 4
+Test
```
Comment (chad-daniels) [inline README.md:34]:
```suggestion
Test requested changes
```

**Review by chad-daniels — CHANGES_REQUESTED** [review]
requested changes

**Comment by chad-daniels** [comment]
Github hi

**Comment by bert-griffin** [comment]
And Berth is here

**Pull request closed without merging**